cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -673,6 +672,37 @@ instance FromCBOR ProtocolParametersUpdate where
         <*> fromCBOR
         <*> fromCBOR
 
+instance ToJSON ProtocolParametersUpdate where
+  toJSON ref =
+    object $
+      ( ("protocolVersion" ..=? protocolUpdateProtocolVersion ref)
```

Comment:
If we have `Generic` instance available here already, why not use `genericToJSON`? 😃